Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt

```diff
@@ -0,0 +1,68 @@
+package com.braintreepayments.demo
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import com.braintreepayments.api.PayPalMessagingListener
+import com.braintreepayments.api.PayPalMessagingLogoType
+import com.braintreepayments.api.PayPalMessagingOfferType
+import com.braintreepayments.api.PayPalMessagingRequest
+import com.braintreepayments.api.PayPalMessagingTextAlignment
+import com.braintreepayments.api.PayPalMessagingView
+
+class PayPalMessagingFragment : BaseFragment(), PayPalMessagingListener {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_paypal_messaging, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val payPalMessagingRequest = PayPalMessagingRequest(
+            amount = 2.0,
+            placement = null,
+            offerType = PayPalMessagingOfferType.PAY_LATER_LONG_TERM,
+            buyerCountry = "US",
+            logoType = PayPalMessagingLogoType.PRIMARY,
+            textAlignment = PayPalMessagingTextAlignment.CENTER,
+            color = null
+        )
+
+        var payPalMessagingView = PayPalMessagingView(braintreeClient, requireActivity())
+        payPalMessagingView.payPalMessagingListener = this
+        payPalMessagingView.start(payPalMessagingRequest)
+        payPalMessagingView.layoutParams = ViewGroup.LayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            ViewGroup.LayoutParams.MATCH_PARENT
+        )
+
+        val messagingView: LinearLayout = view.findViewById(R.id.content)
+        messagingView.addView(payPalMessagingView)
+    }
+
+    override fun onPayPalMessagingClick() {
+        print("User clicked on the PayPalMessagingView")
+    }
+
+    override fun onPayPalMessagingApply() {
+        print("User is attempting to apply for PayPal Credit")
+    }
+
+    override fun onPayPalMessagingLoading() {
+        print("Loading PayPalMessagingView")
+    }
+
+    override fun onPayPalMessagingSuccess() {
+        print("PayPalMessagingView displayed to user")
+    }
+
+    override fun onPayPalMessagingFailure(error: Exception) {
+        print("PayPalMessagingView returned the error:" + error.message)
+    }
```

Comment:
We might want to use `println()` here instead of `print()`.

`println()` will add a new line character to the end of the message: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/println.html

Reply:
Done: https://github.com/braintree/braintree_android/pull/915/commits/35b1b76d31195efd27639d320fbaa58265a52f7c